specs-go/annotations.go

```diff
@@ -0,0 +1,25 @@
+package specs-go
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+
+
+
+/*Annotations annotations
+
+swagger:model annotations
+*/
+type Annotations struct {
+  
+  
+  MapStringString
```

Comment:
I know this is generated code, but this should be `type Annotations map[string]string`.